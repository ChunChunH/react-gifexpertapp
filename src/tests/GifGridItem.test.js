import React from 'react';
import {shallow} from "enzyme";
import GifGridItem from "../components/GifGridItem"

describe('pruebas en gifgriditem', () => {
    test('debe de mostrarse correctamente el componente', () => {

        const wrapper = shallow(<GifGridItem/>)

        expect (wrapper).toMatchSnapshot();

    })
    
})
