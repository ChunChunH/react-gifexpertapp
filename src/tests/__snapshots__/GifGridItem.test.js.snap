// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en gifgriditem debe de mostrarse correctamente el componente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
